source/fem-dem/gls_sharp_navier_stokes.cc

```diff
@@ -1424,7 +1447,9 @@ GLSSharpNavierStokesSolver<dim>::output_field_hook(DataOut<dim> &data_out)
   data_out.add_data_vector(this->present_solution, *levelset_postprocessor);
   Vector<float> cell_cuts(this->triangulation->n_active_cells());
   Vector<float> cell_overconstrained(this->triangulation->n_active_cells());
-
+  Vector<float> cell_has_singular_dof(this->triangulation->n_active_cells());
```

Comment:
I wouldn't keep this here since the code is not supposed to have a singular cell. This was only a tool to find the bug.